src/component/note-container/index.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import uuid from 'uuid/v1';
+
+import NoteList from '../note-list';
+import NoteCreateForm from '../note-create-form';
+
+class NoteContainer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showErrors: true
+    }
+
+    this.noteCreate = this.noteCreate.bind(this);
+    this.noteRemove = this.noteRemove.bind(this);
+    this.noteUpdate = this.noteUpdate.bind(this);
+  }
+
+  noteCreate(note) {
+    note.id = uuid();
+    let {app} = this.props;
+    app.setState(prevState => ({
+      notes: prevState.notes.concat([note])
+    }));
+  }
+
+  noteRemove(note) {
+    let {app} = this.props;
+    app.setState(prevState => ({
+      notes: prevState.notes.filter((item) => {
+        return item.id !== note.id;
+      })
+    }));
+  }
+
+  noteUpdate(note) {
+    let {app} = this.props;
+    app.setState(prevState => ({
+      notes: prevState.notes.map((item) => {
+        return item.id === note.id ? note : item;
+      })
+    }));
+  }
+
+  render() {
+    let {app} = this.props;
+
+    return (
+      <section className='note-container'>
+        <div>
+          <NoteCreateForm handleSubmit={this.noteCreate} />
```

Comment:
In order to pass "props" we are supposed to pass it with the name we are accessing in the chield node, in this case, may be you want to use noteCreate instead of handleSubmit.